build-telemetry-ffi.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0. This product includes software developed
+# at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+set -eu
+
+destdir="$1"
+
+mkdir -v -p "$destdir/include/datadog" "$destdir/lib/pkgconfig" "$destdir/cmake"
```

Comment:
⬇️  This is quite the big bash blob to duplicate, and it's so similar to the profiling one. May be worth taking a stab at having a single one for both telemetry and profiling.

Reply:
I think we should too, but want to sync with you folks first before making more changes to the build process. Lets handle this in a separate PR

Reply:
Sounds extremely reasonable 👍